src/Nancy.Swagger.Annotations/SwaggerObjects/AnnotatedResponse.cs

```diff
@@ -9,7 +9,8 @@ namespace Nancy.Swagger.Annotations.SwaggerObjects
 {
     public class AnnotatedResponse : global::Swagger.ObjectModel.Response
     {
-        public int StatusCode { get; set; }
+        private int StatusCode { get; set; }
+        public int GetStatusCode() => StatusCode;
```

Comment:
This is a silly workaround to get `StatusCode` to stop showing up in the JSON output. Before this change, it looks like:

```
"responses": {
    "200": {
        "StatusCode": 200,
        "description": "OK",
        "schema": {
```
That's the StatusCode in the output I'm talking about. I had to make it a private property to filter out, but it's referenced publicly, so I made the Get method for it.